File: PL033.cpp
#include <iostream>
#include <bits/stdc++.h>
using namespace std;

vector<int> findElements(int a[], int n)
{
    vector<int> v;
    for (int i = 0; i < n; i++)
    {
        v.push_back(a[i]);
    }

    sort(v.begin(), v.end());

    v.pop_back();
    v.pop_back();

    return v;
}

int main()
{
    int n = 6;

    int arr[] = {2, 4, 53, 32, 24, 46};
    vector<int> v;
    v = findElements(arr, n);
    for (int i : v)
    {
        cout << i << " ";
    }
    return 0;
}

File: PL033.java
import java.util.ArrayList;
import java.util.List;

public class PL033 {
    public static void main(String[] args) {
        int n = 6;

        int arr[] = { 2, 4, 53, 32, 24, 46 };

        List<Integer> list = new ArrayList<>();
        list = findElements(arr, n);

        for (Integer i : list) {
            System.out.println(i);
        }
    }

    public static List<Integer> findElements(int[] arr, int n) {
        List<Integer> list = new ArrayList<>();

        for (int i = 0; i < n; i++) {
            list.add(arr[i]);
        }

        list.sort(null);

        list.remove(n - 1);
        list.remove(n - 2);

        return list;
    }
}
